app/res/layout/calendar_widget.xml

```diff
@@ -0,0 +1,174 @@
+        android:orientation="horizontal">
+
+        <ImageButton
+            android:id="@+id/prevmonthbutton"
+            android:layout_width="0dp"
+            android:layout_height="45dp"
+            android:background="@drawable/icon_chevron_left_primary"
+            android:layout_weight=".1"
+            android:layout_gravity="center" />
+
+        <TextView
+            android:id="@+id/currentmonth"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:textStyle="bold"
+            android:layout_weight=".2"
+            android:layout_gravity="center"
+            android:gravity="center_horizontal" />
+
+        <ImageButton
+            android:id="@+id/nextmonthbutton"
+            android:layout_width="0dp"
+            android:layout_height="45dp"
+            android:layout_weight=".1"
+            android:background="@drawable/icon_chevron_right_primary"
+            android:layout_gravity="center" />
+
+        <ImageButton
+            android:id="@+id/prevyearbutton"
+            android:layout_width="0dp"
+            android:layout_height="45dp"
+            android:layout_marginLeft="25dp"
+            android:layout_weight=".1"
+            android:background="@drawable/icon_chevron_left_primary"
+            android:layout_gravity="center" />
+
+        <TextView
+            android:id="@+id/currentyear"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_centerVertical="true"
+            android:textStyle="bold"
+            android:layout_weight=".2"
+            android:layout_gravity="center"
+            android:gravity="center_horizontal" />
+
+        <ImageButton
+            android:id="@+id/nextyearbutton"
+            android:layout_width="0dp"
+            android:layout_height="45dp"
+            android:background="@drawable/icon_chevron_right_primary"
+            android:layout_weight=".1"
+            android:layout_gravity="center" />
+
+    </LinearLayout>
+
+    <LinearLayout
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:gravity="center_horizontal"
+        android:paddingTop="6dp"
+        android:layout_marginLeft="10dp">
+
+        <TextView
+            android:layout_width="0dp"
+            android:textStyle="bold"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:layout_gravity="center_horizontal"
+            android:id = "@+id/day1"/>
+
+        <TextView
+            android:layout_width="0dp"
+            android:textStyle="bold"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:layout_gravity="center_horizontal"
+            android:id = "@+id/day2"/>
+
+        <TextView
+            android:id = "@+id/day3"

```

Comment:
Same comment as above re:spaces (I won't keep commenting in all the places I see it)